contrib/guix/manifest.scm

```diff
@@ -15,13 +15,14 @@
              (gnu packages mingw)
              (gnu packages pkg-config)
              ((gnu packages python) #:select (python-minimal))
-             ((gnu packages python-build) #:select (python-tomli))
+             ((gnu packages python-build) #:select (python-tomli python-poetry-core))
```

Comment:
Do these new dependencies change what is needed for a no-substitutes/boostrap build?

Reply:
Other than the dependency itself, I don't think so. The `pypropject-build-system` is part of python itself, and IIRC `python-poetry-core` is pure python as well with no dependencies.